docs/backend/controlpanels.md

```diff
@@ -0,0 +1,43 @@
+---
+html_meta:
+  "description": ""
+  "property=og:description": ""
+  "property=og:title": ""
+  "keywords": ""
```

Comment:
Please fill in these details, per guidance:

https://6-dev-docs.plone.org/contributing/authors.html#html-and-open-graph-metadata